crates/starknet-server/tests/test_estimate_fee.rs

```diff
@@ -354,6 +356,70 @@ mod estimate_fee_tests {
         assert_eq!(balance_after_sufficient, vec![increase_amount]);
     }
 
+    #[tokio::test]
+    async fn message_available_if_estimation_panics() {
+        let devnet = BackgroundDevnet::spawn().await.expect("Could not start Devnet");
+
+        // get account
+        let (signer, account_address) = devnet.get_first_predeployed_account().await;
+        let account = Arc::new(SingleOwnerAccount::new(
+            devnet.clone_provider(),
+            signer,
+            account_address,
+            CHAIN_ID,
+            ExecutionEncoding::Legacy,
+        ));
+
+        // get class
+        let contract_artifact: SierraClass = load_json(CAIRO_1_BALANCE_CONTRACT_SIERRA_PATH);
```

Comment:
so there is a `get_events_contract_in_sierra_and_compiled_class_hash()` function and I refactor it to `get_contract_in_sierra_and_compiled_class_hash()` https://github.com/0xSpaceShard/starknet-devnet-rs/commit/1f5e010254659c50688baa27fb709369846a399b maybe this code can be replaced with that

Reply:
I changed the name, but the implementation is (almost) the same: https://github.com/0xSpaceShard/starknet-devnet-rs/pull/247/commits/5d177fb6e861be028865fa4373184f7676c5760e
I applied it in other places in my code too